docs/general/attribute-requirement-level.md

```diff
@@ -48,7 +42,25 @@ For example, [Database semantic convention](../database/README.md) references
 `network.transport` attribute defined in [General attributes](./README.md) with
 `Conditionally Required` level on it.
 
-## Required
+## Requirement Levels
+
+The below table seeks to summarise the behaviour of attributes on signals,
+by providing a quick overview.
+In the scenario where an instrumentation has not implemented an opt-in or recommended attribute,
+the configuration option should either not exist or it should operate as a noop.
+The same is the case for conditionally required attributes where the conditions or instructions are not satisfied.
+
+| Level | Included by Default | Included via Config | Excluded via Config |
```

Comment:
```suggestion
| Level | Included by default | Can be included via config | Can be disabled via config |
```

Maybe?

Reply:
or since we use Opt-in opt-out, maybe we use that instead of "included/disabled"?